src/radiation/radiation.cpp

```diff
@@ -364,81 +376,90 @@ void ablate::radiation::Radiation::UpdateCoordinates(PetscInt ipart, Virtualcoor
 }
 
 PetscReal ablate::radiation::Radiation::FaceIntersect(PetscInt ip, Virtualcoord* virtualcoord, PetscFVFaceGeom* faceGeom) const {
-    PetscReal ldotn = (virtualcoord[ip].xdir * faceGeom->normal[0]) + (virtualcoord[ip].ydir * faceGeom->normal[1]) + (virtualcoord[ip].zdir * faceGeom->normal[2]);
+    //!<(planeNormal.dot(planePoint) - planeNormal.dot(linePoint)) / planeNormal.dot(lineDirection.normalize())
+    PetscReal ldotn = 0.0;
```

Comment:
@owenmylotte I think we were using uninitialized data when 1 or 2 Dim.  Does this change make sense?

Reply:
I'm not sure if the switch makes sense to me because 2D needs to add x and y components while 3D needs to add x, y, and z. Will this use all of the components in 3D and I'm misunderstanding?

Reply:
Ah, I think I was missing the fall through. I haven't seen that before.

Reply:
You don't really need the fall through, but c/c++ does fall through by default (probably a bad design).  So you can annotate your code with fall through because most modern compilers now warn you about fall through if you don't annotate it.

Reply:
Yes that looks good to me.